src/frontends/pytorch/src/transforms/quantize_dequantize_replacer.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "quantize_dequantize_replacer.hpp"
+
+#include <memory>
+#include <utility>
+
+#include "openvino/core/rt_info.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "utils.hpp"
+#include "utils_quantize.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+using namespace ov::opset10;
+
+QuantizeDequantizeReplacer::QuantizeDequantizeReplacer() {
+    auto dequantize_node = ov::pass::pattern::wrap_type<ov::frontend::pytorch::QuantizedPtNode>();
+
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto dequantize_node = cast_quantized_fw_node(m.get_match_root(), "aten::dequantize");
+        if (!dequantize_node)
+            return false;
+
+        ov::pass::NodeRegistry rg;
+        std::shared_ptr<ov::Node> quantize_input, dequantize_input, scale, zero_point, axis;
+
+        auto quantize_node = dequantize_node->input_value(0).get_node_shared_ptr();
+        if (auto quantize_per_tensor = cast_quantized_fw_node(quantize_node, "aten::quantize_per_tensor")) {
+            quantize_input = quantize_per_tensor->input_value(0).get_node_shared_ptr();
+            scale = quantize_per_tensor->input_value(1).get_node_shared_ptr();
+            zero_point = quantize_per_tensor->input_value(2).get_node_shared_ptr();
+            auto const dtype_node = quantize_per_tensor->input_value(3).get_node_shared_ptr();
+            auto const const_dtype_node = std::dynamic_pointer_cast<opset10::Constant>(dtype_node);
```

Comment:
what if it is not constant? Should we handle the error for this case somehow?

Reply:
Not applicable in the newest version